apps/sms/views/conversation/js/conversation.js

```diff
@@ -2248,18 +2244,25 @@ var ConversationView = {
     this.container.classList.remove('hide');
   },
 
-  // FIXME/bug 983411: phoneNumber not needed.
-  simSelectedCallback: function conv_simSelected(phoneNumber, cardIndex) {
```

Comment:
:o Nope, I was trying to fix bug 983411 here. as phoneNumber was redundant. PhoneNumber argument is of no use, i guess. Anyway i will fix it in that bug @azasypkin